packages/trader/package.json

```diff
@@ -60,6 +60,7 @@
         "html-loader": "^1.3.2",
         "html-webpack-plugin": "^5.0.0-beta.5",
         "html-webpack-tags-plugin": "^2.0.17",
+        "history": "^5.0.0",
```

Comment:
just checking) `npm run bootstrap:dev` didn't re-generate package-lock.json, right?

Reply:
yeah i ran both bootstrap and bootstrap:dev. no new package.lock